src/plugins/irc-events/ctcp.js

```diff
@@ -21,14 +21,20 @@ module.exports = function(irc, network) {
 		chan.pushMessage(client, msg);
 	});
 
-	irc.on("ctcp request", function(data) {
+	irc.on("ctcp request", (data) => {
 		switch (data.type) {
-		case "PING":
-			var split = data.message.split(" ");
+		case "PING": {
+			const split = data.message.split(" ");
 			if (split.length === 2) {
 				irc.ctcpResponse(data.nick, "PING", split[1]);
 			}
 			break;
 		}
+		case "SOURCE": {
+			const source = "https://www.github.com/thelounge/lounge";
```

Comment:
We could remove `.git` from `repository.url` in `package.json` and then we could just put it in here :)